Source/Details/ASDataController.mm

```diff
@@ -485,7 +486,11 @@ - (void)updateWithChangeSet:(_ASHierarchyChangeSet *)changeSet
     _initialReloadDataHasBeenCalled = YES;
   }
   
-  dispatch_group_wait(_editingTransactionGroup, DISPATCH_TIME_FOREVER);
+  NSTimeInterval transactionQueueFlushDuration;
```

Comment:
Add an initialization, = 0.0f;  This would be initialized to 0 automatically if an instance variable, but in this case it's technically needed.